Tests/FunctionalTests/AEPEdgeFunctionalTests.swift

```diff
@@ -21,7 +21,9 @@ import XCTest
 class AEPEdgeFunctionalTests: FunctionalTestBase {
     private let event1 = Event(name: "e1", type: "eventType", source: "eventSource", data: nil)
     private let event2 = Event(name: "e2", type: "eventType", source: "eventSource", data: nil)
-    private let exEdgeInteractUrl = URL(string: FunctionalTestConst.EX_EDGE_INTERACT_URL_STR)! // swiftlint:disable:this force_unwrapping
+    private let exEdgeInteractProdUrl = URL(string: FunctionalTestConst.EX_EDGE_INTERACT_PROD_URL_STR)! // swiftlint:disable:this force_unwrapping
+    private let exEdgeInteractPreProdUrl = URL(string: FunctionalTestConst.EX_EDGE_INTERACT_PRE_PROD_URL_STR)! // swiftlint:disable:this force_unwrapping
+    private let exEdgeInteractIntegrationUrl = URL(string: FunctionalTestConst.EX_EDGE_INTERACT_INTEGRATION_URL_STR)! // swiftlint:disable:this force_unwrapping
```

Comment:
Should we disable this rule in `Tests/TestUtils/.swiftlint.yml` ?

Reply:
I would like to disable that rule, but I'd prefer to do it in a dedicated task/PR so that we can go through all the newly disabled rules and remove all the comment annotations throughout all our tests. Otherwise, I think we'll end up with many old unneeded `swiftlint:disable` comments.

I can create a task if you'd like.